docs/editor/workspaces.md

```diff
@@ -18,25 +18,25 @@ The concept of a workspace enables VS Code to:
 * Store and restore UI state associated with that workspace (for example, the files that are opened).
 * Selectively enable or disable extensions only for that workspace.
 
-You may see the terms "folder" and "workspace" used interchangeably in VS Code documentation, issues, and community discussions. Think of a workspace as the root of a project that has extra VS Code knowledge and capabilities.
+You may see the terms "folder" and "workspace" used interchangeably in VS Code documentation, issues, and community discussions. You may also see the term "project" in VS Code documentation and in the documentation of other tools, such as IDEs, that VS Code users may also use.  VS Code does not have a definite concept of "project". However for the loose definition of "project" as a "logical collection of remote and local filesystem items, configuration, and settings", you can think of a workspace as the root of a project that has extra VS Code knowledge and capabilities.
```

Comment:
I'd rather not introduce "project" here along with "folder" and "workspace". VS Code uses the term "project" to just mean "the thing you are working on".
I agree it would be good to mention that VS Code doesn't have the concept of a "project' like other IDEs or build systems. This mention could be a Note and leave this paragraph as is.

Reply:
Same feelings